### DISCUSSION TITLE
怎么添加一个组件到文档

### DISCUSSION THREAD

**Discussion by lixiaofa** [youzan/vant#11495] [Q&A]
比如：
![2559580e551b2c30aec21b1e6c44cd4](https://user-images.githubusercontent.com/23230215/212856618-db9da427-1060-4d39-8d9c-30f4feaa7313.png)

将sku组件添加到文档

**Comment by chenjiahan** [comment]
在 [vant.config.mjs](https://github.com/youzan/vant/blob/main/packages/vant/vant.config.mjs) 里配置。
